kit.yml

```diff
@@ -74,3 +74,15 @@ credentials:
     users/blacksmith:   {password: random 64}
 
     registry:           {password: random 64}
+
+  shield-backups:
+    shield:             {password: random 64}
+
+provided:
+  shield-backups:
+    shield:
+      keys:
```

Comment:
Ideally these should be prompted for in the hooks/new script. You can either use a password-style prompt (where the input is obscured) by following [this example](https://github.com/genesis-community/blacksmith-genesis-kit/blob/develop/hooks/new#L56-L57) or [prompt for the value in open text](https://github.com/genesis-community/blacksmith-genesis-kit/blob/develop/hooks/new#L51-L55) and then [set them in safe](https://github.com/genesis-community/blacksmith-genesis-kit/blob/develop/hooks/new#L58)